.github/workflows/semantic-pr-title.yml

```diff
@@ -0,0 +1,16 @@
+name: "Semantic PR title"
+
+on:
+  pull_request:
```

Comment:
Using `pull_request` for testing.
This will need to be changed to `pull_request_target` before merging so it can use the config from target branch for forked PR's.